lesson-02-assignment

```diff
@@ -0,0 +1,93 @@
+#Question 1
+    #create courses 1
+    new_course = CodingClass.all
+    new_course.each do |x|
+    Course.create(coding_class_id:x.id, trimester_id:8, max_enrollment:25)
+    end
+
+    #create courses 2
+    CodingClass.all.each do |x|
+    Course.create(coding_class_id: x.id, trimester_id: 8, max_enrollment: 25)
+    End
+
+#Question 2
+    #create student
+    Student.create(first_name:"Tristan", last_name:"White", email: "Twhite@Gmail.com")
+
+    #create enrollment 
+    Enrollment.create(course_id:44,student_id:51)
+
+    # assign to mentor with less that 2 students
+    # create avaliable mentors
+    available_mentors = []
+    Mentor.all.each  do |x|
+    count =  MentorEnrollmentAssignment.where(mentor_id:x.id).count
+    if count <= 2 
+        available_mentors << x 
+        puts "id##{x.id} #{x.first_name} only has #{count} students"
+    else
+        puts "this mentor has too many students"
+    end
+    end
+    puts available_mentors.pluck(:id,:first_name)
+
+    #Create new MentorEnrollmentAssignment 
+    MentorEnrollmentAssignment.create(mentor_id:available_mentors.last.id,enrollment_id: Enrollment.last.id)
```

Comment:
Instead of relying on `.last` you can save a variable with the `enrollment` in it.